components/perf-tools/scorep/SPECS/scorep.spec

```diff
@@ -113,6 +137,10 @@ make -C build-score CC=$CC V=1 %{?_smp_mflags}
 
 make V=1 %{?_smp_mflags}
 
+%check
+
+make check
+
```

Comment:
Compiler setup missing. PAPI module must be loaded or one test in the `make check` suite fails.